packages/unmock/src/index.ts

```diff
@@ -0,0 +1,4 @@
+import unmock from "unmock-core";
+
+export * from "unmock-core";
+export default unmock;
```

Comment:
Maybe plain
```
export * from "unmock-core";
```
might suffice? I'd rather make the default explicit as I've seen it behave in weird ways 🤔

Reply:
I second that decision. Prefer verbosity over implicitness (<- that does not feel like a proper word in English)